src/sentry/api/endpoints/group_events.py

```diff
@@ -87,7 +87,9 @@ def _get_events_snuba(self, request: Request, group, environments, query, tags,
 
         snuba_filter.conditions.append(["event.type", "!=", "transaction"])
 
-        data_fn = partial(eventstore.get_events, referrer="api.group-events", filter=snuba_filter)
+        data_fn = partial(
+            eventstore.get_unfetched_events, referrer="api.group-events", filter=snuba_filter
```

Comment:
We should probably only use `eventstore.get_unfetched_events` if `not full`. I'm not sure if the unfetched events will work as expected when used with `EventSerializer`?